Automattic/wp-calypso
# Direct Jetpack Backup Plugin to plans on auth

#### Changes proposed in this Pull Request

* Re-enables Jetpack Backup to go to plan selection from auth

#### Testing instructions



* Create a JN site with the Jetpack Plugin and the Jetpack Backup plugin
* Add `define( 'CALYPSO_ENV', 'development' );` to the wp-config.php
* Add `define( 'JETPACK_SHOULD_NOT_USE_CONNECTION_IFRAME', true );`
* Pull this branch locally and setup a local calypso dev environment
* Navigate to the Jetpack Backup plugin and connect to Jetpack
* Ensure the flow navigates to a plans page and not back at the plugin
